Packs/DNSDB/pack_metadata.json

```diff
@@ -1,16 +1,25 @@
 {
-  "name": "Farsight DNSDB",
-  "description": "Query Farsight DNSDB service",
-  "support": "xsoar",
-  "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
-  "url": "https://www.paloaltonetworks.com/cortex",
-  "email": "",
-  "created": "2020-04-14T00:00:00Z",
-  "categories": [
-    "Data Enrichment & Threat Intelligence"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
+    "name": "Farsight DNSDB",
+    "description": "Query Farsight DNSDB service",
+    "support": "partner",
+    "currentVersion": "2.0.0",
+    "author": "Farsight Security, Inc.",
+    "url": "https://github.com/dnsdb",
```

Comment:
The URL needs to be something more official say: https://www.farsightsecurity.com/